site/src/components/WorkspaceSchedule/WorkspaceSchedule.tsx

```diff
@@ -60,7 +60,7 @@ export const Language = {
         return "Workspace is shutting down"
       } else {
         const timezone = schedule ? extractTimezone(schedule) : dayjs.tz.guess()
-        return deadline.tz(timezone).format("HH:mm A")
+        return deadline.tz(timezone).format("hh:mm A")
```

Comment:
```suggestion
        return deadline.tz(timezone).format("hh:mm A z")
```

Reply:
@johnstcn what do you think of adding the timezone here?

Reply:
It could work, I could also foresee complaints if someone travels to a different timezone and has to do the mental computation adjustment. (As someone who does this every day, I just deal with it 😆 )

Reply:
So is the preference to always default to browser?

Reply:
I think that makes sense!